readme.md

```diff
@@ -4,9 +4,9 @@
 [![Build Status](https://travis-ci.org/financialforcedev/orizuru-auth.svg?branch=master)](https://travis-ci.org/financialforcedev/orizuru-auth)
 
 Orizuru authentication is [Express](http://expressjs.com/)-compatible authentication
-middleware for [Node.js](http://nodejs.org/). It is aimed at users of the [Orizuru](https://www.npmjs.com/package/@financialforcedev/orizuru) framework, but can also be used standalone. It is aimed at users of the Salesforce Identity Provider.
+middleware for [Node.js](http://nodejs.org/). It is aimed at users of the [Orizuru](https://www.npmjs.com/package/@financialforcedev/orizuru) framework, but can also be used standalone. It is aimed at users of the Salesforce Identity Provider, although it can be used with other Identity Providers.
 
-In a Salesforce context, using both of these middlewares in combination will establish the following.
+In a Salesforce context, using the middlewares in combination will establish the following.
```

Comment:
'using the middlewares in combination' - 'using all of the middlewares' ?

Reply:
Sorted this out.